api/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.jboss.weld</groupId>
 		<artifactId>weld-parent</artifactId>
-		<version>31</version>
+		<version>36</version>
```

Comment:
I agree with @johnament, I'd like to fix that before the release of 2.0 if possible